### ISSUE TITLE
New pypi release

### ISSUE THREAD

**Issue by allanderek** [jarus/flask-fillin#10]
Hello. The last update to the repository was two years ago, this gave a much required fix for Python 3. Do you think we could have a new pypi release? I know the update is not much, but it's a show-stopper for Python 3 users.

**Comment by MarcUpsher** [comment]
Yep, I'm running into the same issue. Github version is spot on, but Pypi still has the older release that breaks in Python 3.
